predicators/datasets/vlm_input_data.py

```diff
@@ -1,17 +1,245 @@
 """Functions to create offline demonstration data by leveraging VLMs."""
 
+import ast
```

Comment:
now that there's prompting happening inside of this file, can we change the filename to make it sound a little more "active"? it sounds like just datastructures right now